### PR TITLE
Add flakiness to kokoro

### PR DESCRIPTION
**Pull request by steventk-g**
Adds necessary sponge tags to use kokoro for flakiness and greenness reporting.

### CODE REVIEW

File: .kokoro/build.sh [hunk 1]
```diff
@@ -44,6 +44,34 @@ pip install --user https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torch
...
+export_to_sponge_config() {
+  local key="${1}"
+  local value="${2}"
+  printf "${key},${value}\n" >> "${CUSTOM_SPONGE_CONFIG}"
+}
+
+flakiness()
+{
+  export_to_sponge_config "ng3_sponge_url" \
+    "https://fusion2.corp.google.com/invocations/${KOKORO_BUILD_ID}"
```
Comment (miladm) [inline .kokoro/build.sh:65]: How can we skip adding internal URLs to OSS code?

**Review by miladm — CHANGES_REQUESTED** [review]

File: .kokoro/build.sh (hunk 1 above)
Comment (steventk-g) [reply .kokoro/build.sh]: I'll test to see if things work without this field. It doesn't seem like this is strictly required based on the documentation.

Comment (steventk-g) [reply .kokoro/build.sh]: Looks like the field with a link is unnecessary. Removed.

**Comment by vanbasten23** [comment]
I wonder how the metrics generated by Kokoro different from the one generated from bazel. Or is it bazel won't generate the flakiness/greeness metrics?

**Comment by vanbasten23** [comment]
btw, can you point me to the kokoro documentation?

**Review by miladm — APPROVED** [review]

**Pull request merged**